### DISCUSSION TITLE
Doesn't want to process the application

### DISCUSSION THREAD

**Discussion by Michael-Kolpakov** [community/community#88446] [GitHub Education] [answered]
### Hi everyone,

When I fill out all the required information in GitHub Education, upload my student ID, and click the "Process my application" button, nothing happens. The button becomes inactive, but the progress bar with processing does not appear and does not redirect me anywhere. There will be no hints or pop-up messages. Just waiting doesn't help, it goes on forever.

**Comment by afeuerstein** [comment, marked as answer]
Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320
(#88401)

**Comment by Michael-Kolpakov** [comment]
> Seems to be the same problem as here: https://github.com/orgs/community/discussions/88401#discussioncomment-8148320 (#88401)

Thank you so much. Your link helped me solve my problem :)
